src/components/Tabs/Tabs.js

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from "react";
+import Tab from "./Tab";
+import StyledTabs from "./Tabs.styled";
+
+const Tabs = (props) => {
+	const initializeState = (tabs) => {
+		const initObj = {};
+		tabs.forEach(({ props }) => {
+			if (props.active) {
+				initObj[props.eventKey] = props.active ? true : false;
+			}
+		});
+		return initObj;
+	};
+
+	const init = initializeState(props.children);
+	const [activeTab, setActiveTab] = useState(init);
+
+	const handleTabChange = (targetTab) => {
+		const newState = { [targetTab]: true };
+		setActiveTab(newState);
+	};
+
+	const tabList = props.children.map((tab, index) => {
+		const {
+			props: { title, eventKey, active, disabled },
+		} = tab;
+		return (
+			<Tab
+				key={index}
+				handleTabChange={handleTabChange}
+				active={active}
+				eventKey={eventKey}
+				title={title}
+				disabled={disabled}
+			></Tab>
+		);
+	});
+
+	const renderTabContent = () => {
+        // tutaj jest PROBLEM z propsami bo muszę się przebijać przez wszsytko
+		return props.children.map((child, index) => {
+            const { props: { eventKey, children } } = child;
+            const { props } = children;
+			for (let key in activeTab) {
+				if (key === eventKey) {
+					return <div key={index}>{props.children}</div>;
+				}
+			}
+		});
```

Comment:
tutaj bym tylko sprawdzał czy `activeTab === eventKey` i tyle ;)

Reply:
w linijce 29 zostawiłem komentarz bo jestem przekonany że coś tam pomieszałem z propsami, miałem problem żeby je wydobyć. poskracałem to przez destrukturyzację ale w pewnym momencie doszło do tego że miałem w kodzie coś takiego:
`const value = props.children.props.children`. skąd mogło się to wziąć?